src/Database/Bolt/Connection/Type.hs

```diff
@@ -62,4 +62,9 @@ data Request = RequestInit { agent :: Text
              | RequestReset
              | RequestDiscardAll
              | RequestPullAll
-  deriving (Eq, Show)
\ No newline at end of file
+  deriving (Eq, Show)
+
+data Cypher = Cypher
+  { cypherQuery :: Text
+  , cypherParams :: Map Text Value
+  } deriving Show
```

Comment:
I'm not sure, that `Connection.Type` is a best place for this type.

Reply:
Any preferences of where to place it?